contracts/http/context.go

```diff
@@ -25,6 +25,8 @@ type Context interface {
 	Context() context.Context
 	// WithValue add value associated with key in context
 	WithValue(key any, value any)
+
```

Comment:
Pls add comment like other methods.

Reply:
Add sort methods, the `WithContext` should be on top of `WithValue`.